CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Bug Fixes
 
+1. [#5340](https://github.com/influxdata/chronograf/pull/5340): Allow logging out when using Oauth
```

Comment:
i put this as a bug fix for `1.7.17`. I think we should keep this as a patch fix so people can log out without upgrading to 1.8.x (even though it shouldn't be breaking).